Sources/Swift/Tools/SentryLogOutput.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+@objcMembers
+class SentryLogOutput: NSObject {
```

Comment:
`m`: I believe we only use this in Swift, so we can make it Swift only.
```suggestion
class SentryLogOutput {
```